e2e/launcher/browsers.js

```diff
@@ -7,16 +7,5 @@ export const browsersForLocal = [
 ];
 
 export const browsersForCI = [
-	'chrome:headless',
-	'firefox:headless'
```

Comment:
Do we no longer want to run firefox and/or edge?

Reply:
'firefox:headless' doesn't exist